From 225503380d861a6a8bcfb883f01886e0d084c7bf Mon Sep 17 00:00:00 2001
From: Hasan Ahmad <haahmad@blackberry.com>
Date: Fri, 17 May 2013 15:48:53 -0400
Subject: [PATCH 01/61] Update test app with correct cordova.js references

local:///chrome/cordova.js to local:///cordova.js

Reviewed by Bryan Higgins <bhiggins@blackberry.com>
Tested by Tracy Li <tli@blackberry.com>
---
 test/test-app/src/automatic/SpecRunner.htm         |    2 +-
 test/test-app/src/experimental/webviews/child.htm  |    2 +-
 test/test-app/src/experimental/webviews/index.htm  |    2 +-
 test/test-app/src/manual/ContextMenu/index.htm     |    2 +-
 .../test-app/src/manual/DynamicInjection/index.htm |    2 +-
 test/test-app/src/manual/InvocationTest/index.htm  |    2 +-
 .../manual/framework-split/calendarComposer.htm    |    2 +-
 .../src/manual/framework-split/calendarPicker.htm  |    2 +-
 .../src/manual/framework-split/cameraCard.htm      |    2 +-
 .../src/manual/framework-split/emailComposer.htm   |    2 +-
 .../src/manual/framework-split/filePickerCard.htm  |    2 +-
 .../src/manual/framework-split/icsViewerCard.htm   |    6 +--
 test/test-app/src/manual/framework-split/main.htm  |    2 +-
 .../src/manual/framework-split/mediaplayer.htm     |    2 +-
 test/test-app/src/manual/framework/SpecRunner.htm  |    2 +-
 .../src/manual/html5/geolocation_testing.htm       |    4 +-
 .../test-app/src/manual/html5/getUserMediaTest.htm |    2 +-
 test/test-app/src/manual/html5/main.htm            |   40 ++++++++++----------
 test/test-app/src/manual/html5/onload_Testing.htm  |    2 +-
 test/test-suite/Apps/Calendar/index.html           |    4 +-
 test/test-suite/Apps/Card/index.htm                |    2 +-
 test/test-suite/Apps/ConnectionJPPS/index.htm      |    2 +-
 test/test-suite/Apps/CreateContact/index.htm       |    2 +-
 test/test-suite/Apps/DisableWebSecurity/index.html |    2 +-
 test/test-suite/Apps/ExternalStartup/index.html    |    4 +-
 .../Apps/InvocationInterruption/index.htm          |    2 +-
 test/test-suite/Apps/NoAccess/index.html           |    2 +-
 test/test-suite/Apps/Payment/index.html            |    2 +-
 test/test-suite/Apps/PerimeterEvent/index.html     |    2 +-
 .../Apps/SubfolderStartupPage/a/index.html         |    2 +-
 test/test-suite/Apps/TestBgColor/index.htm         |    2 +-
 test/test-suite/Apps/Toast/index.htm               |    2 +-
 test/test-suite/Apps/WildcardDomain/index.html     |    2 +-
 33 files changed, 57 insertions(+), 57 deletions(-)

diff --git a/test/test-app/src/automatic/SpecRunner.htm b/test/test-app/src/automatic/SpecRunner.htm
index 3aa0523..38b9e65 100644
--- a/test/test-app/src/automatic/SpecRunner.htm
+++ b/test/test-app/src/automatic/SpecRunner.htm
@@ -19,7 +19,7 @@
     <link rel="stylesheet" type="text/css" href="../css/jasmine.css">
     <script type="text/javascript" src="../js/jasmine.js"></script>
     <script type="text/javascript" src="../js/jasmine-html.js"></script>
-    <script type="text/javascript" src="local:///chrome/cordova.js"></script>
+    <script type="text/javascript" src="local:///cordova.js"></script>
     <script type="text/javascript">
         var jasmineEnv = jasmine.getEnv(),
             htmlReporter = new jasmine.HtmlReporter(),
diff --git a/test/test-app/src/experimental/webviews/child.htm b/test/test-app/src/experimental/webviews/child.htm
index 0b40edf..a138400 100644
--- a/test/test-app/src/experimental/webviews/child.htm
+++ b/test/test-app/src/experimental/webviews/child.htm
@@ -1,6 +1,6 @@
 <html>
     <head>
-        <script src="local:///chrome/cordova.js"></script>
+        <script src="local:///cordova.js"></script>
         <style type="text/css">
             body {color: white; background: black;}
         </style>
diff --git a/test/test-app/src/experimental/webviews/index.htm b/test/test-app/src/experimental/webviews/index.htm
index 52baa4c..ba0861b 100644
--- a/test/test-app/src/experimental/webviews/index.htm
+++ b/test/test-app/src/experimental/webviews/index.htm
@@ -1,7 +1,7 @@
 <html>
     <head>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <script src="local:///chrome/cordova.js"></script>
+        <script src="local:///cordova.js"></script>
     </head>
     <body>
         <button onclick="document.location.href='local:///experimental/index.htm'">Back</button><br />
diff --git a/test/test-app/src/manual/ContextMenu/index.htm b/test/test-app/src/manual/ContextMenu/index.htm
index 64c4770..7bbd16a 100755
--- a/test/test-app/src/manual/ContextMenu/index.htm
+++ b/test/test-app/src/manual/ContextMenu/index.htm
@@ -17,7 +17,7 @@
 <html>
     <head>
         <title>Context Menu API test</title>
-        <script src="local:///chrome/cordova.js" type="text/javascript"></script>
+        <script src="local:///cordova.js" type="text/javascript"></script>
         <script src="script.js" type="text/javascript"></script>
         <style type="text/css">
             Button {
diff --git a/test/test-app/src/manual/DynamicInjection/index.htm b/test/test-app/src/manual/DynamicInjection/index.htm
index a1244d5..5c8bcd8 100755
--- a/test/test-app/src/manual/DynamicInjection/index.htm
+++ b/test/test-app/src/manual/DynamicInjection/index.htm
@@ -30,7 +30,7 @@
     </head>
     <body bgcolor="Teal">
         <h1 align="center">Dynamic WebWorks Injection</h1>
-        <button onclick=inject('local:///chrome/cordova.js')>Inject WebWorks</button>
+        <button onclick=inject('local:///cordova.js')>Inject WebWorks</button>
         <br />
         <br />
         <br />
diff --git a/test/test-app/src/manual/InvocationTest/index.htm b/test/test-app/src/manual/InvocationTest/index.htm
index f62b62a..c9d25e0 100644
--- a/test/test-app/src/manual/InvocationTest/index.htm
+++ b/test/test-app/src/manual/InvocationTest/index.htm
@@ -2,7 +2,7 @@
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
 <title>Contexts Are Awesome!</title>
-<script src="local:///chrome/cordova.js"></script>
+<script src="local:///cordova.js"></script>
 <script src="js/app.js"></script>
 <link rel="stylesheet" href="css/style.css" type="text/css" media="screen" />
 
diff --git a/test/test-app/src/manual/framework-split/calendarComposer.htm b/test/test-app/src/manual/framework-split/calendarComposer.htm
index 7f6dfd7..c883022 100644
--- a/test/test-app/src/manual/framework-split/calendarComposer.htm
+++ b/test/test-app/src/manual/framework-split/calendarComposer.htm
@@ -2,7 +2,7 @@
 <head>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>calendarComposer card manual tests</title>
-    <script src="local:///chrome/cordova.js" type="text/javascript"></script>
+    <script src="local:///cordova.js" type="text/javascript"></script>
     <script type="text/javascript">
         function invokeCard(options) {
             blackberry.invoke.card.invokeCalendarComposer(options, function () {
diff --git a/test/test-app/src/manual/framework-split/calendarPicker.htm b/test/test-app/src/manual/framework-split/calendarPicker.htm
index 9b32662..5858a73 100644
--- a/test/test-app/src/manual/framework-split/calendarPicker.htm
+++ b/test/test-app/src/manual/framework-split/calendarPicker.htm
@@ -2,7 +2,7 @@
 <head>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>calendarPicker card manual tests</title>
-    <script src="local:///chrome/cordova.js" type="text/javascript"></script>
+    <script src="local:///cordova.js" type="text/javascript"></script>
     <script type="text/javascript">
         function invokeCard(options) {
             blackberry.invoke.card.invokeCalendarPicker(options, function (data) {
diff --git a/test/test-app/src/manual/framework-split/cameraCard.htm b/test/test-app/src/manual/framework-split/cameraCard.htm
index 9ec00ea..d035824 100644
--- a/test/test-app/src/manual/framework-split/cameraCard.htm
+++ b/test/test-app/src/manual/framework-split/cameraCard.htm
@@ -4,7 +4,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>blackberry.invoke.card manual tests</title>
 </head>
-<script src="local:///chrome/cordova.js" type="text/javascript"></script>
+<script src="local:///cordova.js" type="text/javascript"></script>
 <body>
 
     <script type="text/javascript">
diff --git a/test/test-app/src/manual/framework-split/emailComposer.htm b/test/test-app/src/manual/framework-split/emailComposer.htm
index 47860ed..d64c510 100644
--- a/test/test-app/src/manual/framework-split/emailComposer.htm
+++ b/test/test-app/src/manual/framework-split/emailComposer.htm
@@ -2,7 +2,7 @@
 <head>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>emailComposer card manual tests</title>
-    <script src="local:///chrome/cordova.js" type="text/javascript"></script>
+    <script src="local:///cordova.js" type="text/javascript"></script>
     <script type="text/javascript">
         function invokeCard(options) {
             blackberry.invoke.card.invokeEmailComposer(options, function () {
diff --git a/test/test-app/src/manual/framework-split/filePickerCard.htm b/test/test-app/src/manual/framework-split/filePickerCard.htm
index 063b63c..31527fc 100644
--- a/test/test-app/src/manual/framework-split/filePickerCard.htm
+++ b/test/test-app/src/manual/framework-split/filePickerCard.htm
@@ -4,7 +4,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>blackberry.invoke.card manual tests</title>
 </head>
-<script src="local:///chrome/cordova.js" type="text/javascript"></script>
+<script src="local:///cordova.js" type="text/javascript"></script>
 <body>
 
     <script type="text/javascript">
diff --git a/test/test-app/src/manual/framework-split/icsViewerCard.htm b/test/test-app/src/manual/framework-split/icsViewerCard.htm
index 2918cce..a53aa5f 100644
--- a/test/test-app/src/manual/framework-split/icsViewerCard.htm
+++ b/test/test-app/src/manual/framework-split/icsViewerCard.htm
@@ -4,7 +4,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>blackberry.invoke.card manual tests</title>
 </head>
-<script src="local:///chrome/cordova.js" type="text/javascript"></script>
+<script src="local:///cordova.js" type="text/javascript"></script>
 <body>
 
     <script type="text/javascript">
@@ -24,8 +24,8 @@
                             return val;
                         }
                     }
-                    return segment(2) + "-" + segment(1) + "-" + segment(1) + "-" + segment(1) +  "-" + segment(3); 
-                } 
+                    return segment(2) + "-" + segment(1) + "-" + segment(1) + "-" + segment(1) +  "-" + segment(3);
+                }
 
                 function gotWriter(fileWriter) {
                     fileWriter.onwriteend = function (e) {
diff --git a/test/test-app/src/manual/framework-split/main.htm b/test/test-app/src/manual/framework-split/main.htm
index 9ea0a59..5b2b0ad 100644
--- a/test/test-app/src/manual/framework-split/main.htm
+++ b/test/test-app/src/manual/framework-split/main.htm
@@ -3,7 +3,7 @@
 	<head>
 	    <meta name="viewport" content="width=device-width, initial-scale=1" />
 		<title>WebWorks Smoke Test for BB10</title>
-		<script src="local:///chrome/cordova.js" type="text/javascript"></script>
+		<script src="local:///cordova.js" type="text/javascript"></script>
         <script src="local:///js/jquery-1.7.2.min.js"></script>
         <script src="local:///js/jquery.mobile-1.1.1.min.js"></script>
         <link rel="stylesheet" href="local:///css/jquery.mobile-1.1.1.min.css" />
diff --git a/test/test-app/src/manual/framework-split/mediaplayer.htm b/test/test-app/src/manual/framework-split/mediaplayer.htm
index c019bba..330e7a9 100644
--- a/test/test-app/src/manual/framework-split/mediaplayer.htm
+++ b/test/test-app/src/manual/framework-split/mediaplayer.htm
@@ -4,7 +4,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>blackberry.invoke.card manual tests</title>
 </head>
-<script src="local:///chrome/cordova.js" type="text/javascript"></script>
+<script src="local:///cordova.js" type="text/javascript"></script>
 <body>
 
     <script type="text/javascript">
diff --git a/test/test-app/src/manual/framework/SpecRunner.htm b/test/test-app/src/manual/framework/SpecRunner.htm
index 705ad1d..0211b23 100644
--- a/test/test-app/src/manual/framework/SpecRunner.htm
+++ b/test/test-app/src/manual/framework/SpecRunner.htm
@@ -7,7 +7,7 @@
     <link rel="stylesheet" type="text/css" href="../../css/jasmine.css">
     <script type="text/javascript" src="../../js/jasmine.js"></script>
     <script type="text/javascript" src="../../js/jasmine-html.js"></script>
-    <script type="text/javascript" src="local:///chrome/cordova.js"></script>
+    <script type="text/javascript" src="local:///cordova.js"></script>
 
     <script type="text/javascript">
         var jasmineEnv = jasmine.getEnv();
diff --git a/test/test-app/src/manual/html5/geolocation_testing.htm b/test/test-app/src/manual/html5/geolocation_testing.htm
index 24a224a..7b77118 100644
--- a/test/test-app/src/manual/html5/geolocation_testing.htm
+++ b/test/test-app/src/manual/html5/geolocation_testing.htm
@@ -7,7 +7,7 @@
 
         <title>test geolocation</title>
 
-        <script type="text/javascript" src="local:///chrome/cordova.js"></script>
+        <script type="text/javascript" src="local:///cordova.js"></script>
 
         <style type="text/css">
             input[type='text'] {
@@ -130,7 +130,7 @@
             <input type="button" value="Browser"  onclick="invokeBrowser('http://benwerd.com/lab/geo.php');" />
         </section>
 
-        <section>	
+        <section>
 	    <a href="main.htm">Back</a>
         </section>
 
diff --git a/test/test-app/src/manual/html5/getUserMediaTest.htm b/test/test-app/src/manual/html5/getUserMediaTest.htm
index a7f4f4a..2d53b8b 100644
--- a/test/test-app/src/manual/html5/getUserMediaTest.htm
+++ b/test/test-app/src/manual/html5/getUserMediaTest.htm
@@ -1,7 +1,7 @@
 ﻿<html>
 <head>
     <title>getUserMedia</title>
-	<script src="local:///chrome/cordova.js" type="text/javascript"></script>
+	<script src="local:///cordova.js" type="text/javascript"></script>
 </head>
 <body>
 	<div data-bb-type="screen">
diff --git a/test/test-app/src/manual/html5/main.htm b/test/test-app/src/manual/html5/main.htm
index 1be5de5..2e7f947 100644
--- a/test/test-app/src/manual/html5/main.htm
+++ b/test/test-app/src/manual/html5/main.htm
@@ -3,7 +3,7 @@
 	<head>
 	    <meta name="viewport" content="width=device-width, initial-scale=1" />
 		<title>WebWorks Smoke Test for BB10</title>
-		<script src="local:///chrome/cordova.js" type="text/javascript"></script>
+		<script src="local:///cordova.js" type="text/javascript"></script>
         <script src="local:///js/jquery-1.7.2.min.js"></script>
         <script src="local:///js/jquery.mobile-1.1.1.min.js"></script>
         <link rel="stylesheet" href="local:///css/jquery.mobile-1.1.1.min.css" />
@@ -11,43 +11,43 @@
             function do_LocalImageTest() {
                 document.location.href = "local_Image_Testing.htm";
             }
-			
+
 			function do_LocalCSSTest() {
                 document.location.href = "local:///manual/html5/local_CSS_Testing.htm";
             }
-			
+
 			function do_LocalJSFileTest() {
 				document.location.href = "local_JSFile_Testing.htm";
 			}
-			
+
 			function do_AjaxTest() {
                 document.location.href = "local:///manual/html5/local_Ajax_Testing.htm";
             }
-			
+
 			function do_WebDataBaseTest() {
                 document.location.href = "local:///manual/html5/html5db.htm";
             }
-			
+
 			function do_WebWorkerTest() {
                 document.location.href = "local:///manual/html5/webworker_Testing.htm";
             }
-	
+
 			function do_LocalStorageTest() {
                 document.location.href = "local:///manual/html5/localStorage.htm";
             }
-			
+
 			function do_ManifestCacheTest() {
                 document.location.href = "local:///manual/html5/iframeManifestCache.htm";
             }
-			
+
 			function do_GeolocationTest() {
                 document.location.href = "local:///manual/html5/geolocation_testing.htm";
             }
-			
+
 		    function do_onload_Test() {
 				document.location.href = "local:///manual/html5/onload_Testing.htm";
 			}
-			
+
 			function do_flash_video_audio_Test() {
                 alert("If enable-flash feature is specified, a flash page should load");
                 document.location.href = "http://www.homestarrunner.com/sbemail103.html";
@@ -59,12 +59,12 @@
             function do_input_Test() {
                 document.location.href = "local:///manual/html5/input.html";
             }
-			
+
         </script>
 	</head>
 	<body>
-        
-        <input id="backButton" type="button" value="Back" onclick="document.location.href = 'local:///index.htm';" /> 
+
+        <input id="backButton" type="button" value="Back" onclick="document.location.href = 'local:///index.htm';" />
 
 	    <p> This WebWorks Smoke Test includes the following tests: </p>
 		<ol>
@@ -85,7 +85,7 @@
         <p>
             <input id="Button2" type="button" value="Local Image Test" onclick="return do_LocalImageTest()" /></p>
 		<p>
-            <input id="Button3" type="button" value="Local CSS Test" onclick="return do_LocalCSSTest()" /></p>	
+            <input id="Button3" type="button" value="Local CSS Test" onclick="return do_LocalCSSTest()" /></p>
 		<p>
             <input id="Button4" type="button" value="Local JS File Test" onclick="return do_LocalJSFileTest()" /></p>
 	    <p>
@@ -99,15 +99,15 @@
         <p>
             <input id="Button9" type="button" value="Manifest Cache Test" onclick="return do_ManifestCacheTest()" /></p>
         <p>
-            <input id="Button10" type="button" value="Geolocation Test" onclick="return do_GeolocationTest()" /></p>						
+            <input id="Button10" type="button" value="Geolocation Test" onclick="return do_GeolocationTest()" /></p>
 		<p>
             <input id="Button14" type="button" value="onload Testing" onclick="return do_onload_Test()" /></p>
 		<p>
-            <input id="Button15" type="button" value="Flash Player Audio & Video Test" onclick="return do_flash_video_audio_Test()" /></p>				
+            <input id="Button15" type="button" value="Flash Player Audio & Video Test" onclick="return do_flash_video_audio_Test()" /></p>
 		<p>
-            <input id="Button16" type="button" value="Web Inspector Test" onclick="return do_web_inspector_Test()" /></p>					
+            <input id="Button16" type="button" value="Web Inspector Test" onclick="return do_web_inspector_Test()" /></p>
             <p>
-                <input id="Button17" type="button" value="Input Test" onclick="return do_input_Test()" /></p>					
+                <input id="Button17" type="button" value="Input Test" onclick="return do_input_Test()" /></p>
             <p>
                 <input id="Button178" type="button" value="Input File Types" onclick="document.location ='local:///manual/html5/input_types.html';" />
             </p>
@@ -115,6 +115,6 @@
                 <input id="Button19" type="button" value="Get User Media" onclick="document.location ='local:///manual/html5/getUserMediaTest.htm';" />
             </p>
     </ol>
-	
+
 	</body>
 </html>
diff --git a/test/test-app/src/manual/html5/onload_Testing.htm b/test/test-app/src/manual/html5/onload_Testing.htm
index 6003d78..98dc900 100644
--- a/test/test-app/src/manual/html5/onload_Testing.htm
+++ b/test/test-app/src/manual/html5/onload_Testing.htm
@@ -2,7 +2,7 @@
 <head>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>onload Testing</title>
-    <script src="local:///chrome/cordova.js"></script>
+    <script src="local:///cordova.js"></script>
 	<script type="text/javascript">
 	function displaySmokeTestDetails()
 	{
diff --git a/test/test-suite/Apps/Calendar/index.html b/test/test-suite/Apps/Calendar/index.html
index 2a21b69..842119b 100644
--- a/test/test-suite/Apps/Calendar/index.html
+++ b/test/test-suite/Apps/Calendar/index.html
@@ -2,7 +2,7 @@
 <head>
 	<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
 	<link rel="stylesheet" href="textarea.css" type="text/css" media="screen" />
-	<script type="text/javascript" src="local:///chrome/cordova.js"></script>
+	<script type="text/javascript" src="local:///cordova.js"></script>
 	<script type="text/javascript" src="textarea.js"></script>
 	<script type="text/javascript">
 		var cal,
@@ -182,4 +182,4 @@
 	<input style="width: 100px; height: 100px; font-size: 200%" type="button" onclick="removeSingleOccurrence()" value="Remove Single Occurrence">
 	<textarea></textarea>
 </body>
-</html>
\ No newline at end of file
+</html>
diff --git a/test/test-suite/Apps/Card/index.htm b/test/test-suite/Apps/Card/index.htm
index 5aa81bf..770b651 100755
--- a/test/test-suite/Apps/Card/index.htm
+++ b/test/test-suite/Apps/Card/index.htm
@@ -17,7 +17,7 @@
 <html>
 	<head>
 		<title>InvokedEventCard</title>
-		<script src="local:///chrome/cordova.js" type="text/javascript"></script>
+		<script src="local:///cordova.js" type="text/javascript"></script>
 		<script src="script.js" type="text/javascript"></script>
 	</head>
 	<body bgcolor="Teal">
diff --git a/test/test-suite/Apps/ConnectionJPPS/index.htm b/test/test-suite/Apps/ConnectionJPPS/index.htm
index f582e7f..e61b82b 100755
--- a/test/test-suite/Apps/ConnectionJPPS/index.htm
+++ b/test/test-suite/Apps/ConnectionJPPS/index.htm
@@ -2,7 +2,7 @@
 <head>
     <title>Connection API</title>
 	<script type="text/javascript" src="script.js"></script>
-	<script type="text/javascript" src="local:///chrome/cordova.js"></script>
+	<script type="text/javascript" src="local:///cordova.js"></script>
 </head>
 <body>
 	<button onclick="alert(blackberry.connection.type)">blackberry.connection.type</button><br /><br />
diff --git a/test/test-suite/Apps/CreateContact/index.htm b/test/test-suite/Apps/CreateContact/index.htm
index f16362c..afdcc2d 100755
--- a/test/test-suite/Apps/CreateContact/index.htm
+++ b/test/test-suite/Apps/CreateContact/index.htm
@@ -17,7 +17,7 @@
 <html>
 	<head>
 		<title>CreateContacts</title>
-		<script src="local:///chrome/cordova.js" type="text/javascript"></script>
+		<script src="local:///cordova.js" type="text/javascript"></script>
 		<link rel="stylesheet" href="textarea.css" type="text/css" media="screen" />
 		<script type="text/javascript" src="textarea.js"></script>
 		<script src="script.js" type="text/javascript"></script>
diff --git a/test/test-suite/Apps/DisableWebSecurity/index.html b/test/test-suite/Apps/DisableWebSecurity/index.html
index 2bdcaf5..3435ce8 100644
--- a/test/test-suite/Apps/DisableWebSecurity/index.html
+++ b/test/test-suite/Apps/DisableWebSecurity/index.html
@@ -7,7 +7,7 @@
     <link rel="stylesheet" type="text/css" href="css/jasmine.css">
     <script type="text/javascript" src="js/jasmine.js"></script>
     <script type="text/javascript" src="js/jasmine-html.js"></script>
-    <script type="text/javascript" src="local:///chrome/cordova.js"></script>
+    <script type="text/javascript" src="local:///cordova.js"></script>
 
     <!-- include spec files here... -->
     <script type="text/javascript" src="spec/disableWebSecurityTests.js"></script>
diff --git a/test/test-suite/Apps/ExternalStartup/index.html b/test/test-suite/Apps/ExternalStartup/index.html
index 4005daa..d646bb5 100644
--- a/test/test-suite/Apps/ExternalStartup/index.html
+++ b/test/test-suite/Apps/ExternalStartup/index.html
@@ -3,7 +3,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Local External Startup Page</title>
 
-    <script type="text/javascript" src="local:///chrome/cordova.js"></script>
+    <script type="text/javascript" src="local:///cordova.js"></script>
     <script type="text/javascript">
         function postBlackberryId(e) {
             e.source.postMessage(blackberry.app.id, event.origin);
@@ -12,7 +12,7 @@
         document.addEventListener('webworksready', function () {
             // Post to parent to indicate we are ready
             window.addEventListener('message', postBlackberryId, false);
-            parent.postMessage('wwready', "*"); 
+            parent.postMessage('wwready', "*");
         });
     </script>
 </head>
diff --git a/test/test-suite/Apps/InvocationInterruption/index.htm b/test/test-suite/Apps/InvocationInterruption/index.htm
index c3e6066..4752445 100755
--- a/test/test-suite/Apps/InvocationInterruption/index.htm
+++ b/test/test-suite/Apps/InvocationInterruption/index.htm
@@ -17,7 +17,7 @@
 <html>
 	<head>
 		<title>Invocation Interruption</title>
-		<script src="local:///chrome/cordova.js" type="text/javascript"></script>
+		<script src="local:///cordova.js" type="text/javascript"></script>
 		<script src="script.js" type="text/javascript"></script>
 	</head>
 	<body bgcolor="Teal">
diff --git a/test/test-suite/Apps/NoAccess/index.html b/test/test-suite/Apps/NoAccess/index.html
index c4b1ed6..826f5fc 100644
--- a/test/test-suite/Apps/NoAccess/index.html
+++ b/test/test-suite/Apps/NoAccess/index.html
@@ -7,7 +7,7 @@
   <link rel="stylesheet" type="text/css" href="css/jasmine.css">
   <script type="text/javascript" src="js/jasmine.js"></script>
   <script type="text/javascript" src="js/jasmine-html.js"></script>
-  <script type="text/javascript" src="local:///chrome/cordova.js"></script>
+  <script type="text/javascript" src="local:///cordova.js"></script>
 
   <!-- include spec files here... -->
   <script type="text/javascript" src="spec/noAccessTests.js"></script>
diff --git a/test/test-suite/Apps/Payment/index.html b/test/test-suite/Apps/Payment/index.html
index ce4adf1..f47a558 100755
--- a/test/test-suite/Apps/Payment/index.html
+++ b/test/test-suite/Apps/Payment/index.html
@@ -17,7 +17,7 @@
 <html>
     <head>
         <title>Payment</title>
-        <script src="local:///chrome/cordova.js" type="text/javascript"></script>
+        <script src="local:///cordova.js" type="text/javascript"></script>
         <script src="script.js" type="text/javascript"></script>
     </head>
     <body bgcolor="Teal">
diff --git a/test/test-suite/Apps/PerimeterEvent/index.html b/test/test-suite/Apps/PerimeterEvent/index.html
index 711b400..4047fba 100755
--- a/test/test-suite/Apps/PerimeterEvent/index.html
+++ b/test/test-suite/Apps/PerimeterEvent/index.html
@@ -17,7 +17,7 @@
 <html>
 	<head>
 		<title>Perimeter Events</title>
-		<script src="local:///chrome/cordova.js" type="text/javascript"></script>
+		<script src="local:///cordova.js" type="text/javascript"></script>
 		<script src="script.js" type="text/javascript"></script>
 	</head>
 	<body bgcolor="Teal">
diff --git a/test/test-suite/Apps/SubfolderStartupPage/a/index.html b/test/test-suite/Apps/SubfolderStartupPage/a/index.html
index 3deaea5..4d5549b 100644
--- a/test/test-suite/Apps/SubfolderStartupPage/a/index.html
+++ b/test/test-suite/Apps/SubfolderStartupPage/a/index.html
@@ -7,7 +7,7 @@
     <link rel="stylesheet" type="text/css" href="../css/jasmine.css">
     <script type="text/javascript" src="../js/jasmine.js"></script>
     <script type="text/javascript" src="../js/jasmine-html.js"></script>
-    <script type="text/javascript" src="local:///chrome/cordova.js"></script>
+    <script type="text/javascript" src="local:///cordova.js"></script>
 
     <!-- include spec files here... -->
     <script type="text/javascript" src="../spec/whitelistSubfolder.js"></script>
diff --git a/test/test-suite/Apps/TestBgColor/index.htm b/test/test-suite/Apps/TestBgColor/index.htm
index 5699db8..8191dff 100755
--- a/test/test-suite/Apps/TestBgColor/index.htm
+++ b/test/test-suite/Apps/TestBgColor/index.htm
@@ -1,7 +1,7 @@
 <html>
 	<head>
 		<title>Page 1</title>
-		<script type="text/javascript" src="local:///chrome/cordova.js"></script>
+		<script type="text/javascript" src="local:///cordova.js"></script>
 	</head>
 	<body bgcolor="#FF9900">
         <div align="center">
diff --git a/test/test-suite/Apps/Toast/index.htm b/test/test-suite/Apps/Toast/index.htm
index 080ca15..00ebcb4 100755
--- a/test/test-suite/Apps/Toast/index.htm
+++ b/test/test-suite/Apps/Toast/index.htm
@@ -17,7 +17,7 @@
 <html>
 	<head>
 		<title>Invoke Toast Messages</title>
-		<script src="local:///chrome/cordova.js" type="text/javascript"></script>
+		<script src="local:///cordova.js" type="text/javascript"></script>
 		<script src="script.js" type="text/javascript"></script>
 	</head>
 	<body bgcolor="Teal">
diff --git a/test/test-suite/Apps/WildcardDomain/index.html b/test/test-suite/Apps/WildcardDomain/index.html
index ae67553..5ab7a13 100644
--- a/test/test-suite/Apps/WildcardDomain/index.html
+++ b/test/test-suite/Apps/WildcardDomain/index.html
@@ -7,7 +7,7 @@
   <link rel="stylesheet" type="text/css" href="css/jasmine.css">
   <script type="text/javascript" src="js/jasmine.js"></script>
   <script type="text/javascript" src="js/jasmine-html.js"></script>
-  <script type="text/javascript" src="local:///chrome/cordova.js"></script>
+  <script type="text/javascript" src="local:///cordova.js"></script>
 
   <!-- include spec files here... -->
   <script type="text/javascript" src="spec/crossOriginWildcard.js"></script>
-- 
1.7.10.msysgit.1

